ga4gh/backend.py

```diff
@@ -57,6 +59,8 @@ def runSearchRequest(
         (object, nextPageToken) pairs, and be able to resume iteration from
         any point using the nextPageToken attribute of the request object.
         """
+        if self.profiler is not None:
```

Comment:
What if we did this in a subclass rather than updating the code for the backend object? So, we just have
```python
class ProfilingBackend(backend.Backend):
      def runSearchRequest(stuff):
             profiler.setrelheap()
             super(ProfilingBackend, self).runSearchRequest(stuff)
             print(profiler.heap())
```
This way, we wouldn't need to touch backend.py at all.